test/lib/docusign_transaction_rooms/paginated_resource_test.rb

```diff
@@ -0,0 +1,11 @@
+require 'test_helper'
+
+module DocusignTransactionRooms
+  class PaginatedResourceTest < Minitest::Test
+
+    def test_something
+      skip('need to figure out what should be tested here instead of via resources using it')
```

Comment:
You could unit test all the public methods.

Reply:
didn't know if that would add any value, figured that they were being hit with the tests on the #all call in the room resource test

Reply:
Then just remove this file